src/status_im/navigation/effects.cljs

```diff
@@ -8,12 +10,27 @@
     [taoensso.timbre :as log]
     [utils.re-frame :as rf]))
 
+(def previous-screen (atom nil))
+
+(defn- set-status-bar-color
+  [theme view-id]
+  (when-not (= @previous-screen :settings-syncing)
```

Comment:
Please can you elaborate, why we still need to store previous-screen?

Reply:
It looks like some kind of bug. I don't think we should close settings screen while opening syncing screen.
(we can confirm this with the design team, but it is somewhat obvious)

Please remove workaround, we can create an issue and address this navigation issue in follow up pr.